src/modules/ekf2/EKF/aid_sources/optical_flow/optical_flow_control.cpp

```diff
@@ -222,13 +223,59 @@ void Ekf::controlOpticalFlowFusion(const imuSample &imu_delayed)
 	}
 }
 
-void Ekf::resetFlowFusion(const flowSample &flow_sample)
+void Ekf::resetFlowFusion(const flowSample &flow_sample, VectorState &H)
 {
+	// dont force hard reset if innovation is acceptable
```

Comment:
IMO this should be the responsibility of the logic calling `resetFlowFusion` to decide whether a reset or a fusion should be done. `resetFlowFusion` should just implement the reset.